scripts/style-checker.sh

```diff
@@ -34,12 +34,13 @@ function setup_root_dir
 
 function astyle_c_check
 {
-    astyle --options="$root_dir/.astylerc" --dry-run "$root_dir/*.c" | grep "Formatted" | tee badly-formatted-files.list | wc -l | while read badly_formatted_c_files
+    astyle --options="$root_dir/.astylerc" "$root_dir/*.c" | grep "Formatted" | tee badly-formatted-files.list | wc -l | while read badly_formatted_c_files
     do
         echo "Number of badly formatted files: $badly_formatted_c_files"
         if [ "$badly_formatted_c_files" == "0" ]; then
             exit 0
         else
+            git diff
             cat badly-formatted-files.list
             exit 1
```

Comment:
So this patch should not be part of this PR  (I'm not sure that my previous comment is at the right place... )